src/Main.elm

```diff
@@ -48,7 +49,7 @@ subscriptions model =
 
 viewDocument : Model -> Browser.Document Msg
 viewDocument model =
-    { title = "[cCc] App title", body = [toUnstyled (view model) ] }
+    { title = "[cCc] App title", body = [toUnstyled (PageTemplate.view { content = Just(view model) }) ] }
```

Comment:
This looks like you don't have elm-format integrated with your editor yet. Want a hand setting that up?

Reply:
yes please!

Reply:
actually don't worry about it, have set it up